README.md

```diff
@@ -17,190 +17,88 @@ By [Fauna, Inc](http://faunadb.com).
 
 ## Inroduction
 
-This repository contains FaunaDB driver for Swift language. It provides high level abstractions that allows us to work with fauna DB efficiently and without the need to deal with rest messages, networking errors, data encoding, decoding and so on.
-
-> working on another language? Take a look at our github account, we provide Fauna DB drivers for many popular languages and many others are coming. Don't hesitate to contact us if the language you are working on is not supported yet. Notice you can always use the [REST API](https://faunadb.com/documentation/rest) directly.
+This repository contains FaunaDB driver for Swift language. It provides high level abstractions that allows us to work with Fauna DB efficiently and without the need to deal with rest messages, networking errors, data encoding, decoding and so on.
```

Comment:
I think the less verbose intro of other drivers: "Swift [FaunaDB](https://faunadb.com/) Driver."  is probably enough :)